### PR TITLE
Adds Team page from navbar, removes template artifacts

### PR DESCRIPTION
**Pull request by michaelwnau**
This PR addresses Issue #82 'Create Team Page.' Employs team assets, insights assets, and call to action.

### CODE REVIEW

File: team.html [hunk 1]
```diff
@@ -6,266 +6,211 @@
...
     <div class="container">
         <div class="row justify-content-center">
-            <div class="col-lg-8">
-                <div class="theme_title team_title_3 text-center">
-                    <span class="theme_span">team</span>
-                    <h2>Meet Our Leadership</h2>
+            <div class="col-lg-6">
+                <div class="theme_title team_title text-center">
+                    <span class="theme_span">Crew</span>
+                    <h2>Meet Your Team<br>at ZCore Group</h2>
```
Comment (bastosmichael) [inline team.html:15]: Our Team?

File: team.html [hunk 2]
```diff
@@ -6,266 +6,211 @@
...
-<section class="zCore_testimonial zCore_testimonial_2" style="background-image: url({{relative_url}}/assets/images/testmonial_bg.jpg);">
-    <div class="zCore_overlay"></div>
-    <div class="container">
-        <div class="row">
-            <div class="col-lg-12">
-                <div class="testimonial_slide_1">
-                    <div class="testimonial_item">
-                        <div class="testimonial_img wow slideInLeft">
-                            <img src="{{relative_url}}/assets/images/testimonial_2.png" alt="">
+<!-- <section class="zCore_testimonial zCore_testimonial_2" style="background-image: url({{relative_url}}/assets/images/testmonial_bg.jpg);">
```
Comment (bastosmichael) [inline team.html:74]: better to remove code than comment it out, google will see and index this stuff so we don't want that.

File: team.html [hunk 3]
```diff
@@ -6,266 +6,211 @@
...
             </div>
         </div>
     </div>
 </section>
-<!-- End zCore_blog section -->
+<!-- End zCore_blog_2 section -->
 <!-- Start call_to_action_3 section -->
-<section class="call_to_action_3 dark_blue section_padding">
+{% include contact.html %}
+<!-- <section class="call_to_action_3 dark_blue section_padding">
```
Comment (bastosmichael) [inline team.html:213]: delete code instead of commenting out.

File: team.html (hunk 1 above)
Comment (michaelwnau) [reply team.html:15]: Message in dev chat

**Review by bastosmichael — APPROVED** [review]

**Pull request merged**